database_admin/migrations/003_triggers.down.sql

```diff
@@ -1,5 +1,3 @@
-DROP TRIGGER system_platform_set_first_reported ON system_platform;
```

Comment:
It doesn't make sense to modify previous migrations. It's needed to create new one. But I would prefer to split this task into two phases. Firstly remove all first_reported info in go code without db updates. After some time running this code, we can create PR to remove it also from db.